src/__tests__/real-world-example/real-world-example.test.tsx

```diff
@@ -17,7 +21,7 @@ describe('real-world-example', () => {
 
     await screen.findByText('Authenticated content');
     expect(
-      screen.getByText('Cookie: SessionId=super-secret')
+      screen.getByText('Cookie: TrackingId=123; SessionId=super-secret')
```

Comment:
Confirming cookies are not overwritten on page navigation. On page render we set only `'TrackingId=123';` and that is correctly propagated to the second page on client side navigation.